test/jdk/java/awt/event/ComponentEvent/TextAreaTextEventTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is the year in which this test was initially written known? If yes, it is good to have that added here.